src/guides/bioinformatics/estimate-clade-frequencies.rst

```diff
@@ -0,0 +1,758 @@
+========================================================
+Estimate frequencies of phylogenetic clades through time
```

Comment:
> Another potential issue is how we should maintain guides like this that we generate directly from a notebook environment.

What about an extension like [nbsphinx or MyST-NB](https://docs.readthedocs.io/en/stable/guides/jupyter.html)? We'd still have to run the notebook to generate new plots but less manual work than maintaining the same content in both `.ipynb` and `.rst`.